sys/console/full/src/console.c

```diff
@@ -467,6 +473,8 @@ console_handle_char(uint8_t byte)
         case ESC:
             esc_state |= ESC_ESC;
             break;
+        default:
+            insert_char(&input->line[cur], byte, end);
```

Comment:
if this fall through is intentional please add a comment (GCC7 will emit warning without it)